docs/guide/user_management/invitations.md

```diff
@@ -0,0 +1,75 @@
+---
+description: Managing user invitations to create an account in the frontend or the Back Office.
+---
+
+# Inviting Users
+
+[[= product_name =]] allows you to create and send invitations to create an account in
+the frontend as customer, the Back Office as employee, or the Corporate Portal as organisation member.
+You can send invitations to individual users or in bulk.
+
+## Roles and Policies
+
+To invite other members to the site or the Back Office User needs to have `User:Invite` permission added to his Role.
+You can limit the ability to invite other members to the specific User Groups
+e.g. Editors, or to the specific Roles within the group e.g. Admin, Buyer.
+
+## Creating and sending invitations
+
+Invitations are created with [InvitationService](https://github.com/ibexa/user/blob/main/src/lib/Invitation/InvitationService.php),
+but sending them requires additional setup.
+[[= product_name =]] provides you with `InvitationSender` interface for sending invitations via email.
```

Comment:
We provide implementation of that interface. (\Ibexa\User\Invitation\MailSender)